### PR TITLE
fix: missing experiment status

### PR DESCRIPTION
**Pull request by g1eny0ung**
Signed-off-by: Yue Yang <g1enyy0ung@gmail.com>

### What problem does this PR solve?

Add missing `Status` field used in the dashboard.

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by YangKeao — DISMISSED** [review]
LGTM

File: cmd/chaos-builder/impl.go [hunk 1]
```diff
@@ -97,6 +97,7 @@ func (in *{{.Type}}) GetChaos() *ChaosInstance {
 		Kind:      Kind{{.Type}},
 		StartTime: in.CreationTimestamp.Time,
 		Action:    "",
+		Status:    string(in.GetStatus().Experiment.Phase),
```
Comment (cwen0) [inline cmd/chaos-builder/impl.go:100]: Why we need use `in.GetStatus()` function here? Can we use `in.Status.xxxx` directly?

File: cmd/chaos-builder/impl.go [hunk 2]
```diff
@@ -97,6 +97,7 @@ func (in *{{.Type}}) GetChaos() *ChaosInstance {
 		Kind:      Kind{{.Type}},
 		StartTime: in.CreationTimestamp.Time,
 		Action:    "",
+		Status:    string(in.Status.ChaosStatus.Experiment.Phase),
```
Comment (cwen0) [inline cmd/chaos-builder/impl.go:100]: `in.Status.Experiment.Phase`. we can omit `ChaosStatus`

Comment (g1eny0ung) [reply cmd/chaos-builder/impl.go]: Got it! Already fixed.

**Review by fewdan — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**